spec/models/restaurant_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+describe Restaurant do
+  let(:restaurant) { Restaurant.new(name: "test", url: "http://hoge.test/", description: "testtest") }
+
+  describe "name" do
+    it "is not valid without name" do
+      restaurant.name = nil
+      expect(restaurant).not_to be_valid
+    end
+
+    it "should have a unique name" do
+      Restaurant.create(name: "test", url: "http://example.com", description: "")
+      expect(restaurant.valid?).to eq(false)
```

Comment:
@yukimurasawa ひとつ上のテストで `be_valid` を使っているのに対して判定方法を変えているのが気になりました 🤔  `.not_to be_valid` として書き方を統一すると、何をテストしているのかがわかりやすいと思いました。
```suggestion
      expect(restaurant).not_to be_valid
```

また、`Restaurant.create` の部分は `before` を使うとよりわかりやすいです！
`before` は「テストの前提条件」の役割になるのでここでテストしたい「すでに同じ名前のRestaurantデータが存在する場合」ということが明示的になります

```ruby
before { Restaurant.create(name: "test", url: "http://example.com", description: "") }

it "should have a unique name" do
  expect(restaurant).not_to be_valid
end
```